packages/widget.api/src/messages/widgetMessages.ts

```diff
@@ -62,11 +82,20 @@ export namespace WidgetMessage {
   export interface EmitEvent<C extends WidgetConfig<C["parameters"]>>
     extends WidgetBaseMessage<"widget.emit-event", Payload.EmitEvent<C>>
   {}
+
+  /**
+   * Slate event payload that the widget sends to the host Foundry UI
+   */
+  export interface EmitSlateEvent<C extends WidgetConfig<C["parameters"]>>
+    extends
+      WidgetBaseMessage<"widget.emit-slate-event", Payload.EmitSlateEvent<C>>
```

Comment:
Wondering if we considered reusing `widget.emit-event` and moving the type union / guards onto the payload itself which may also make it easier to reuse a single `emitEvent` method and have the types block the code from actually sending a wrongly typed workshop vs slate event based on the config